client/statsdb.js

```diff
@@ -3,8 +3,14 @@ const loki = require("lokijs")
 
 // / Wrapper over a lokijs persistent storage to keep track of the stats of executing accounts.
 class StatsDB {
-  constructor(web3) {
-    this.db = new loki("stats.json")
+  constructor(web3, browserDB) {
+    debugger
+    if (browserDB) {
+      this.db = browserDB
+    } else {
+      this.db = new loki("stats.json")
+    }
+
```

Comment:
I think we should elevate the DB choice above eac.js-client. Pass either loki or browserdb or whatever has instert,update,find and get functions.